api/py/ai/chronon/constants.py

```diff
@@ -1,5 +0,0 @@
-from ai.chronon.scheduler.adapters.airflow_adapter import AirflowAdapter
```

Comment:
question: 
does it mean we removed the dependency on Airflow?

Reply:
We just decoupled `JoinBackfill` with `Airflow`. The `AirflowAdapter` or any other customized adapters can be passed through the [orchestrator](https://github.com/airbnb/chronon/pull/799/files#diff-70939bbcad59a3e132ccbbf2b5797512419bd014ea73bdd2ee1c6de879f2012dR124) argument in `JoinBackfill.run()`. This way, we don't have to import adapters in the `JoinBackfill` module.

Reply:
Got it. 
yeah, this is better. Thanks @donghanz